src/looper.ts

```diff
@@ -95,18 +95,17 @@ export function isIDValid(config: Config, ids: string): boolean {
   return true
 }
 
+let isAborted = false
```

Comment:
moving this variable to global scope will cause the value of isAborted to stay `true` after it's first set to true. 

This means that when monika receive new configuration from Symon, it clear probe interval from the previous configuration and the new interval for the new configuration will also get aborted immediately. Or in other word, the next configurations will never run.